modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -4845,7 +4845,7 @@ definition {
 		property test.liferay.virtual.instance = "false";
 		property test.run.type = "single";
 
-		task ("Given a regular role with multi-byte characters in the title") {
+		task ("Given a regular role with multi-byte characters in the name") {
```

Comment:
Please, check if this is done through the UI or API . If UI, then this should be "key".

Reply:
Hello Carlos, we are sending the roleKey in the name property of the API within JSONRole.addRegularRole. 

What do you think?